cmd/kyma/alpha/create/module/module.go

```diff
@@ -64,13 +65,18 @@ Build module modB in version 3.2.1 and push it to a local registry "unsigned" su
 	cmd.Flags().StringVar(&o.Channel, "channel", "stable", "Channel to use for the module template.")
 	cmd.Flags().StringVarP(&o.Token, "token", "t", "", "Authentication token for the given registry (alternative to basic authentication).")
 	cmd.Flags().BoolVarP(&o.Overwrite, "overwrite", "w", false, "overwrites the existing mod-path directory if it exists")
-	cmd.Flags().BoolVar(&o.Insecure, "insecure", false, "Use an insecure connection to access the registry.")
+	cmd.Flags().StringVar(&o.Insecure, "insecure", "", "Use an insecure connection to access the registry.")
```

Comment:
Why was this flag changed to string?

Reply:
Well, perhaps I misunderstood the AC. It's mentioned there: `If a registry has http schema and the --insecure flag is not explicitly set` - Users can set this to false, since it's a boolean. So I wanted a way to detect if the flag was used by the user.

Reply:
I see, with `not set` I meant that it is not true. Since boolean flags are true as soon as they are in the input; `--insecure` and `--insecure=true` are equivalent.

Thus the flag being not set and `--insecure=false` are also equivalent.